.travis.yml

```diff
@@ -1,4 +1,11 @@
 language: java
 jdk: oraclejdk8
 sudo: false
-script: mvn clean deploy
+cache:
+  directories:
+  - $HOME/.m2
```

Comment:
What does "cache" do and why is it necessary?

Reply:
not necessary, but makes the build faster.  https://docs.travis-ci.com/user/caching/#Arbitrary-directories